app/controllers/subjects_controller.rb

```diff
@@ -0,0 +1,74 @@
+class SubjectsController < ApplicationController
+  before_action :set_subject, only: [:show, :edit, :update, :destroy]
+
+  # GET /subjects
+  # GET /subjects.json
```

Comment:
What is the value of having the comments here though? Is the code too confusing to understand? Even as an MVP product, we should limit unnecessary lines of codes and comments so that we have less to maintain in going forward. As we update code, we'd have to ensure the comments that go along with the code is updated as well; hence, there is more to maintain.
Here is the Ruby styleguide's thoughts on comments: https://github.com/bbatsov/ruby-style-guide#no-comments